style.css

```diff
@@ -539,35 +540,41 @@ section {
 
 .popup {
   position: fixed;
-  top: 50%;
+  top: 47%;
   left: 50%;
   transform: translate(-50%, -50%);
   background-color: #fff;
   padding: 22px;
-  border-radius: 8px;
+  border-radius: 16px;
+  border: 1px solid lightgray;
   box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
   z-index: 9999;
-  width: 80vw;
+  width: 78%;
+  height: 83.4%;
   display: flex;
```

Comment:
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but please always try to make your design as close as possible to provided design, if you look at the below-attached table of design comparison, your design needs some adjustments related to spaces around the popUp, colour and many other small issues. I would request you kindly fix your PopUp issue, and make it looks professional and similar to Figma's design. Below I am actually going to tell each and everything in more detail in numeric points.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-31-56](https://github.com/NoelLincoln/mportfolio/assets/66476329/8b39c374-0973-4b4e-ae70-70558d910358)

</td>
<td>

![Screenshot from 2023-07-01 18-33-25](https://github.com/josman227/portfolio.github.io/assets/66476329/daea9a3f-9499-45cc-b772-62d40e5cde85)

</td>
</tr>
</table>

### 🖊️ Improvements need

1. Your `popUp` project **heading** color is not matching with the original design and also needs some space at the bottom to make your secondary information looks more visible.
2. Please fix the size of your *image* it is a little bit bigger than the actual which makes your popUp little looks longer and little space is also needed at the bottom to make it looks beautiful and similar to the original design.
3. Your popUp project description **color** is not matching and needs some spacing too at the bottom. 
4. Please reduce the gap between the skills and also your skills text needs to refactor in terms of styling like reducing either `height` or `width` or `padding`.
5. and 6. Both are buttons, if you look above carefully because your button looks weird and needs to fix by adjusting the spacing, put the icon equally to the text at the **x-axis** properly.

### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-39-57](https://github.com/NoelLincoln/mportfolio/assets/66476329/eaed1c3c-b6ae-4536-9437-9f72f2d0eb8b)

</td>
<td>

![Screenshot from 2023-07-01 18-31-04](https://github.com/josman227/portfolio.github.io/assets/66476329/a0c499f8-00b8-4324-9e0c-44188468f8df)

</td>
</tr>
</table>


### 🖊️ Improvements need

1. Your `popUp` project **heading** color is not matching with the original design. 
2. Please add some space at the top to make your secondary information looks more visible.
3. Please fix the size of your *image* it is a little bit bigger than the actual which makes your popUp little looks longer and little space is also needed at the bottom to make it looks beautiful and similar to the original design.
4. Your popUp project description **color** is not matching and needs some spacing too at the bottom. 
5. Please reduce the gap between the skills and also your skills text needs to refactor in terms of styling like reducing either `height` or `width` or `padding`.
6. and 7. Both buttons look ok on the **Desktop** screen size but need some adjustments like proper spacing and also it collapsing with the *skills*.
8. **Kindly reduce the width of the popUp in your desktop screen-size**.